packages/tools/test/BidirectionalTool_test.js

```diff
@@ -1,6 +1,8 @@
 import * as cornerstone3D from '@cornerstonejs/core';
 import * as csTools3d from '../src/index';
 import * as testUtils from '../../../utils/test/testUtils';
+import * as cornerstoneTools from '@cornerstonejs/tools';
```

Comment:
not a big deal, but just saying for future, let's order the library imports at top and the local ones ('../../etc') after

Reply:
Ok, I will try to be more careful next time. Thanks for the 'heads up'. I am not 100% certain, but I think it might have been VS Code that did this automatically. Funny enough, that import is no longer needed.